spec/cypress/support/step_definitions/common-interaction.js

```diff
@@ -0,0 +1,40 @@
+import { When, Then } from "cypress-cucumber-preprocessor/steps";
+
+const elements = {
+  "cookie consent": '[name="cookies_form[analytics_consent]"]',
+  "cookie banner": ".js-cookie-banner",
+};
```

Comment:
I hate this!

Reply:
I hate this so much I would advocate for making variable names use underscores over spaces.

Reply:
It's less the actual implementation of it I hate and more just the general concept

Reply:
Wow - so much of hate speech - I guess no redemption for cookie banner

Reply:
I can't think of a decent solution for this tbh so let's just leave it for now